app/components/molecules/promotion-card/promotion-card.component.tsx

```diff
@@ -10,16 +16,18 @@ interface PromotionCardProps {
   label: string;
 }
 
-export const PromotionCard: FC<PromotionCardProps> = ({
+export const PromotionCard: FC<PromotionCardProps & CardProps> = ({
```

Comment:
how does this work? 🤔  isnt the same as a interface extension? - I was having this question yesterday

Reply:
This is called a type intersection and it means that PromotionCard expects both the props defined in PromotionCardProps and CardProps. I think it is the equivalent of an interface extension like you mention. I found these sources: [TypeScript Intersection Types](https://www.typescripttutorial.net/typescript-tutorial/typescript-intersection-types/) and [TypeScript: Differences Between Type Aliases and Interfaces](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces).